lib/serverengine/supervisor.rb

```diff
@@ -95,18 +103,32 @@ def create_server(logger)
       @server = @create_server_proc.call(@load_config_proc, logger)
     end
 
+    def stop_internal(stop_graceful)
+      begin
+        _stop(stop_graceful)
+      rescue Errno::EPIPE
+        # nothing to do
+      ensure
+        if @command_pipe
+          @command_pipe.close rescue nil
```

Comment:
is it ok to ignore this error? I think that the thread started at install_signal_handlers won't be stopped unexpectedly if this `close` failed.